util/difgen/gen_dif_listing.py

```diff
@@ -123,5 +124,12 @@ def _get_text_or_empty(element, xpath):
     if inner is None:
         return ""
 
-    # If there's no inner text, return ""
-    return inner.text or ""
+    # Create a string from all subelements
+    text = ET.tostring(inner, encoding="unicode", method="html")
+    # Code highlighting is supported for the tag 'code' use it instead of
+    # 'computeroutput'
+    text = text.replace("<computeroutput>",
+            "<code>").replace("</computeroutput>", "</code>")
+    # Manually remove the enclosing tags
+    text = re.sub("<[^<]+>(.*)</.+>", "\\1", text)
```

Comment:
Is this robust? Is it a static encosing tag, that is emitted?

Reply:
It might not be :)
I think the enclosing tag is always there as it is generated by `tostring` but I did not check for a prove.

Maybe change the regex to apply only to strings which start with `<`?

Reply:
```python
"^<[^<]+>(.*)</.+>$"
```
@wallento your regex expertise is certainly better than mine, should that be enough?

Reply:
I would suggest to operate on the ElementTree instead of fiddling with the text. Which tag are you removing here? div? I can have a look into it later today

Reply:
The tag is always the name of the path with which the element was found.
So for example `<name>dif_alert_handler_lock</name>` for a call with `func_info["name"] = _get_text_or_empty(m, "name")`.

Would appreciate if you will have a look at it! But I am not sure if operating on the ElementTree will be that easy either as this will alter the underlying data (I guess).

But we can also just leave the tag there, I assume no harm will be done by doing that.